### ISSUE TITLE
pip installing to Python 2.7 installs the non 2.7 compatible version 

### ISSUE THREAD

**Issue by selimt** [ivankorobkov/python-inject#73]
It seems like pip installing the latest version of inject on Python 2.7 tries to install version 4.3.1 which is compatible with 3.6+ . Is there a way for it to resolve to the latest 2.7 compatible version?

Thanks.

**Comment by ivankorobkov** [comment]
Hi,

There is a table with supported versions in readme:

Python | Inject Version
-- | --
3.6+ | 4.1+
3.5 | 4.0
< 3.5 | 3.*

You can install a specific version with:
`pip install 'Inject==3.5.4'`

**Comment by selimt** [comment]
Yes but if your setup.py contained the ‘python_requires’ option as specified here:

https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires

then pip would not try to install an incompatible release.

**Comment by ivankorobkov** [comment]
You may add it and make a pull request, if you want :-)
